src/App.tsx

```diff
@@ -70,13 +71,18 @@ const App: React.FC = () => {
     }
   }, [dispatch, user.id]);
 
+  useEffect(() => {
```

Comment:
syns det er litt vanskelig å få en rask forståelse av hva denne faktisk gjør. Personlig liker jeg en kort kommentar som sier noe om hva/hvorfor. Vet samtidig at noen liker å holde kommentarer til et minimum, so do whatever floats your boat

Reply:
tja, jeg er ikke så fan av kommentarer, og synes ikke denne er den verste på funksjonalitet, men kan legge til kommentar om hva som skjer hvis flere synes dette er rart? evt skrive:
`if (user.id && !user.authority) { setShowModal(true)}`?

Reply:
Syns denne syntaxen er helt grei for min del.